app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    My inventory
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    <%= "There are currently #{@inventory.length} ingredients in my inventory!" %>
+  </p>
+  <div class="block">
+    <%= link_to "Add ingredient", new_inventory_path, class: "button is-success" %>
+  </div>
+
+  <div class="columns">
+    <div class="column box">
+      <div class="tags">
+        <% @inventory.each do |item| %>
+          <%= tag.span "", class: "tag is-success" do %>
+            <%= item.ingredient.name.capitalize %>
```

Comment:
See how you are iterating through every inventory item, but also, calling their __ingredient__, this wasn't originally considered in the query, thus will geneate n+1 queries.
Consider adding `includes(:ingredient)` to the @inventory query at the inventory query.
Take a look at [this article](https://engineering.gusto.com/a-visual-guide-to-using-includes-in-rails/) to understand better how this could save up some time :D

Reply:
This include has been included (xD) in the latest commit